pkg/controller/etcdproxy/etcdproxy_controller_test.go

```diff
@@ -32,6 +32,7 @@ import (
 
 	etcdclient "github.com/xmudrii/etcdproxy-controller/pkg/client/clientset/versioned/fake"
 	etcdlisters "github.com/xmudrii/etcdproxy-controller/pkg/client/listers/etcd/v1alpha1"
+	"github.com/xmudrii/etcdproxy-controller/pkg/options"
```

Comment:
This dep strikes me as odd

Reply:
Fixed. Looks like that one caused import cycles as well.